src/System.Diagnostics.Process/tests/Helpers.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit.Sdk;
+
+namespace System.Diagnostics.Tests
+{
+    internal static class Helpers
+    {
+        public static async Task RetryWithBackoff(Action action, int delayInMilliseconds = 10, int times = 10)
+        {
+            for (; times > 0; times--)
+            {
+                try
+                {
+                    action();
+                    return;
+                }
+                catch (XunitException) when (times > 1)
+                {
+                    await Task.Delay(delayInMilliseconds);
+                    delayInMilliseconds *= 2;
```

Comment:
Added. I'm not sure what value to use for the cut-off, though. You mentioned maybe 1 second; @danmosemsft [previously suggested](https://github.com/dotnet/corefx/pull/31827#discussion_r213510986) allowing things to go a bit longer. Right now, I have it set to 10 seconds. Glad to adjust to more or less if that is not a good compromise value.

Reply:
I suggest to leave as is, I have increased so many timeouts on tests that for some reason run slow in official runs 1 in 1000 times because the machine is paging or something.

Reply:
Sounds good to me.